jdeeco-core/src/cz/cuni/mff/d3s/deeco/runtimelog/RuntimeLogger.java

```diff
@@ -0,0 +1,537 @@
+package cz.cuni.mff.d3s.deeco.runtimelog;
```

Comment:
I just had a discussion with the guys and we concluded that it would make more sense to use the runtime logger as an instance. This way, we could combine deeco simulation with UDP-based real-time deeco node in same JVM. Please, do you think you could remove the "static" keyword in the class? Ilias and Filip will make sure that the runtime logger instance is accessible from the DEECo class. Thanks.